linalg/hypre.hpp

```diff
@@ -334,6 +342,7 @@ class HypreParMatrix : public Operator
 
    /// MPI communicator
    MPI_Comm GetComm() const { return A->comm; }
+   void BuildComm() const { hypre_MatvecCommPkgCreate(A); }
```

Comment:
What does this do? Needs some documentation.

Reply:
Was only used once, removed function and used hypre_MatvecCommPkgCreate where needed.